TESTING.md

```diff
@@ -0,0 +1,74 @@
+
+# Requirements
+
+In order to run the full test suite users will need access to a running Manta
```

Comment:
Maybe link to the testing documentation here?

Reply:
Please clarify, this is the testing documentation file.

Reply:
Whoops, I thought I made that comment on another doc. Scratch this.